src/test/kotlin/org/opensearch/replication/MultiClusterRestTestCase.kt

```diff
@@ -421,6 +422,13 @@ abstract class MultiClusterRestTestCase : OpenSearchTestCase() {
     }
 
     protected fun wipeIndicesFromCluster(testCluster: TestCluster) {
+
```

Comment:
From Follower stats API response, do we need to filter any indices? I mean just wanted to confirm that we are iterating over syncing/bootstrapping/failed indices but not stopped ones.

Reply:
follower indices whose status is `REPLICATION NOT IN PROGRESS` are giving `200 OK` response when passed to stopReplication API, If those indices are skipped then DELETE request is failing